WcaOnRails/app/webpacker/components/Panel/Wfc/index.jsx

```diff
@@ -42,19 +38,17 @@ const sections = [
 ];
 
 export default function Wfc() {
-  const { data, loading, error } = useLoadedData(panelWfcPageDataUrl);
+  const { loggedInUserPermissions, loading } = useLoggedInUserPermissions();
 
   if (loading) return <Loading />;
   return (
-    <>
-      {error && <Errored />}
-      <PanelTemplate
-        heading="WFC Panel"
-        sections={sections
-          .filter(
-            (section) => !section.forAtleastSeniorMember || (data?.isAtleastSeniorMember === true),
-          )}
-      />
-    </>
+    <PanelTemplate
+      heading="WFC Panel"
+      sections={sections
+        .filter(
+          (section) => (!section.forAtleastSeniorMember
+              || loggedInUserPermissions.canAccessWfcSeniorStuff()),
```

Comment:
Also, why are you accessing this `canAccess...` thing like it's a function?
It should just be a key in a JSON object(?)

Reply:
Because I was trying to make everything in `useLoggedInUserPermissions` as function, instead of some as values and some as functions. Making all as values is not possible because of `canEditRole` from https://github.com/thewca/worldcubeassociation.org/pull/8593.

Reply:
Why do you need to make everything a function? What goes against making the function a function, and keep other things as simple key-value?

Reply:
Reason is that I felt having all the items of this hook of same type: either all function or all value will make it more clean. I think even @kr-matthews brought your point in some other PR. Anyway, I've changed it to the way you suggested.